sysl2/lib/lang/java.sysl

```diff
@@ -0,0 +1,889 @@
+# sysl.lang.java
+!grammar java:
+    Identifier:
+        IdentifierChars - (Keyword | BooleanLiteral | NullLiteral)
+
+    IdentifierChars:
+        /[[:letter:]][[:letter:][:digit:]]/
```

Comment:
`:letter:` and `:digit:` aren't defined anywhere, right? Also, isn't there a `*` missing?

Reply:
`[:digit:]` is a standard posix regex element https://goo.gl/F9zQKy. `[:letter:]` should have been `[:alpha:]`, but really this is just rough thinking at the moment. It's not intended as a final compilable grammar yet.

Reply:
ok.

Reply:
@anzdaddy - I'm still wondering about the missing `*`, but happy I'm happy for you to merge anyway.